pmg/__init__.py

```diff
@@ -193,7 +193,7 @@ def send_email_with_sendgrid(message):
 
 # if we don't do this in a separate thread, we hang trying to connect to the db
 import pmg.tasks
-Thread(target=pmg.tasks.schedule).start()
+# Thread(target=pmg.tasks.schedule).start()
```

Comment:
is this supposed to be commented out still?

Reply:
Sorry no, committed that by mistake.